statsmodels/genmod/tests/test_glm.py

```diff
@@ -1450,9 +1448,13 @@ def __init__(self):
 
 
 class TestTweediePower2(CheckTweedie):
-    def __init__(self):
+    @classmethod
+    def setupClass(self):
         from .results.results_glm import CpunishTweediePower2
-        super(TestTweediePower2, self).__init__()
+        from statsmodels.datasets.cpunish import load_pandas
+        self.data = load_pandas()
+        self.exog = self.data.exog[['INCOME', 'SOUTH']]
+        self.endog = self.data.endog
```

Comment:
for next time:
one possible trick is to leave these in a method in the super class and call the method directly, which works (IIRC) if the method in the superclass is also a `classmethod`
(could be a `setupClass` or a `getdata` method in super)
super call needs the class twice, IIRC, e.g. `super(TestTweediePower2, TestTweediePower2).get_data()`

("IIRC" means it works but I never remember the details and have to look them up from other examples.)